doc/administrator/mapfile.rst

```diff
@@ -400,6 +400,14 @@ Mapserver allows different forms of legends.
     END
 
 
+Performance imrovement
+----------------------
+
+In the ``LAYER`` we can specify an ``EXTENT`` we can specify an extent to that can be a big
```

Comment:
Adding an ``EXTENT`` parameter in the ``LAYER`` section may significantly improve the performances.